docs/docs/50-roadmap.md

```diff
@@ -0,0 +1,55 @@
+---
+title: Roadmap
+Description: Kargo Roadmap
+---
+
+# Roadmap
+
+Kargo does not currently have a regular release cadance. The plan is to do a few releases, and based on the resluts, create a more formal release cadance.
+
+:::caution
+This roadmap is subject to change at any time, for the most up to date information, please see the [GitHub Project](https://github.com/akuity/kargo/projects)
+:::
+
+# v0.2.0
+
+| Name | Type | Description |
+| ---- | ---- | ----------- |
+| `Freight/Warehouse CRD` | feature | Freight will change from being a property of a `Stage`, to being it's own CRD. A collection of promotable `Freight` will be known as a `Warehouse`. |
+| `Long Lived Promotions` | feature | Support for Pull Requset based promotion, which is indefinite. This will allow the promotion to last as long as the PR is open. |
+| `Kargo Render` | breaking change | `Bookkeeper` to be retooled to be the Kargo specific way to do rendered manifests. |
+
+
+# v0.3.0
+
+| Name | Type | Description |
+| ---- | ---- | ----------- |
+| `Analysis` | feature | Run analysis against a metrics provider to determite application health. This will allow assessing Promotion success based on more than just Argo CD Application state. |
```

Comment:
Ability to run Analysis for the purposes of Freight qualification. This enables users to define additional quality gates (in addition to Argo CD Health) to allow/disallow promotion.